ycsb/main.go

```diff
@@ -206,7 +206,7 @@ func (yw *ycsbWorker) runLoader(n uint64, numWorkers int, thisWorkerNum int, wg
 	if *verbose {
 		fmt.Printf("Worker %d loading %d rows of data\n", yw.workerID, n)
```

Comment:
Not your code, but this statement appears incorrect - worker `yw.workerID` is loading much less than `n` rows of data judging by the loop below